File: sla/sla.c
#include <stdio.h>

int main() {
    double num1, num2, resultado;
    char op;

    printf("Digite o primeiro numero: ");
    scanf("%lf", &num1);  // %lf = double

    printf("Digite o operador (+, -, *, /): ");
    scanf(" %c", &op);    // espaço antes do %c ignora o enter anterior

    printf("Digite o segundo numero: ");
    scanf("%lf", &num2);

    if (op == '+') resultado = num1 + num2;
    else if (op == '-') resultado = num1 - num2;
    else if (op == '*') resultado = num1 * num2;
    else if (op == '/') {
        if (num2 != 0) resultado = num1 / num2;
        else { printf("Erro: divisao por zero!\n"); return 1; }
    } else {
        printf("Operador invalido!\n");
        return 1;
    }

    printf("Resultado: %.2lf\n", resultado); // %.2lf = 2 casas decimais
    return 0;
}

File: sla/sla.ts
import * as readlineSync from 'readline-sync';

let num1 = parseFloat(readlineSync.question("Digite o primeiro numero: "));
let op = readlineSync.question("Digite o operador (+, -, *, /): ");
let num2 = parseFloat(readlineSync.question("Digite o segundo numero: "));

let resultado: number;

if (op === "+") resultado = num1 + num2;
else if (op === "-") resultado = num1 - num2;
else if (op === "*") resultado = num1 * num2;
else if (op === "/") {
    if (num2 !== 0) resultado = num1 / num2;
    else { console.log("Erro: divisao por zero!"); process.exit(1); }
} else { console.log("Operador invalido!"); process.exit(1); }

console.log("Resultado:", resultado);
